drivers/classic/src/display_current_settings.c

```diff
@@ -85,12 +85,12 @@ display_current_settings(int mode)
         fprintf(LOG_DEST, "EQUAL_AREA\t\tFALSE\n");
     }
     fprintf(LOG_DEST, "RESOLUTION\t\t%f\n", global_param.resolution);
-    fprintf(LOG_DEST, "TIME_STEP\t\t%d\n", global_param.dt);
-    fprintf(LOG_DEST, "SNOW_STEP\t\t%d\n", options.SNOW_STEP);
+    fprintf(LOG_DEST, "TIME_STEP\t\t%f\n", global_param.dt);
+    fprintf(LOG_DEST, "SNOW_STEP\t\t%f\n", global_param.snow_dt);
```

Comment:
If these are `double`s (not sure). The format specifier should be `%lf` rather than `%f` (`float`)